O3-1.tex

```diff
@@ -22,37 +22,40 @@
 
 \title{The LSST Data Management System}
 \author{
-Mario~Juric,$^1$
-Jeffrey~Kantor,$^2$
```

Comment:
I'd prefer the force push without the single commit. Otherwise I'll cherry pick (I obviously want you to be credited with the change).